.github/workflows/format_fix_pr.yml

```diff
@@ -22,25 +20,6 @@ jobs:
           # Without this we would be forever waiting on required checks to pass
           token: ${{ secrets.GH_PAT_APPROVINGTON_OCTAVIA }}
 
-      # IMPORTANT! This is necessary to make sure that a status is reported on the PR
-      # even if the workflow is skipped. If we used GitHub Actions filters, the workflow
-      # would not be reported as skipped, but instead would be forever pending.
-      #
-      # I KNOW THIS SOUNDS CRAZY, BUT IT IS TRUE.
-      #
-      # Also, it gets worse
-      #
-      # IMPORTANT! DO NOT CHANGE THE QUOTES AROUND THE GLOBS. THEY ARE REQUIRED.
-      # MAKE SURE TO TEST ANY SYNTAX CHANGES BEFORE MERGING.
-      - name: Get changed files
-        uses: tj-actions/changed-files@v39
-        id: changes
-        with:
-          files_yaml: |
-            format:
-              - '**/*'
-              - '!**/*.md'
-
```

Comment:
@erohmensing I also removed in for `format_check_pr`. I think it's not required anymore as:
* The worfklow runs on all PRs, it's never skipped and the `airbyte-ci format check all` will always report a success or failure.

Reply:
> The workflow runs on all PRs, it's never skipped and the airbyte-ci format check all will always report a success or failure.

it looks to me like the previous logic here is to not run it for markdown files, but for everything else. To be clear, we're deciding to now run it on all PRs, including docs PRs, because it is safer and fast enough, and the manual action is there to help someone who doesn't have airbyte-ci deal with any formatting failures. right?

Reply:
I'd prefer to not bypass format for docs. We might introduce a markdown formatter one day 🤔